Resources/config/dbal.xml

```diff
@@ -91,5 +91,18 @@
             <tag name="console.command" command="doctrine:query:sql" />
         </service>
 
+
+        <service id="doctrine.create_database_command" class="Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand">
+            <argument type="service" id="doctrine" />
+
+            <tag name="console.command" command="doctrine:database:create" />
+        </service>
+
+        <service id="doctrine.drop_database_command" class="Doctrine\Bundle\DoctrineBundle\Command\DropDatabaseDoctrineCommand">
```

Comment:
@dunglas these 2 commands are already registered as services with no args l74-78 above, updating them should fix the build

Reply:
Good catch, I missed that! Thank you!!